libs/ngxtension/click-outside/src/click-outside.ts

```diff
@@ -34,14 +37,16 @@ const [injectDocumentClick] = createInjectionToken(() => {
  * <div (clickOutside)="close()"></div>
  *
  */
-@Directive({ selector: '[clickOutside]', standalone: true })
+@Directive({
+	selector: '[clickOutside]',
+	standalone: true,
+	providers: [provideDocumentClick()],
```

Comment:
This will create a listener for each directive, while the previous method, would create only one listener for the whole document, and each directive will use the same listener, which makes it more performant I think.

Reply:
How can it be more performant if the observer is active all the time? :) I thought about it and figured that using multiple outsideClicks on one view would be rare.

Is memory leak something we should accept?